scripts/download-binaries.sh

```diff
@@ -67,10 +67,25 @@ curl "${curl_args}O" "${golint_url}" \
     && tar xzfP "${golint_tgz}" -C "${dest_dir}" --strip-components=1 "${golint_dir}/golangci-lint" \
     && rm "${golint_tgz}"
 
+gobindata_version="3.1.2"
```

Comment:
Ugh, I didn't realize the extent of go-bindata's messy history. And contrary to my request, there is no binary release available. Given that what appears here is essentially what `go get` does, I think it would be preferable to just use that and perhaps symlink the resulting binary into `./bin` for consistency.

Reply:
@marun This (https://github.com/go-bindata/go-bindata/releases) repo is maintained and they recently cut a new release.

Reply:
As per our offline conversation, please update to `go get` in preference to what appears here.